templates/webapps/galaxy/workflow/run_complete.mako

```diff
@@ -0,0 +1,55 @@
+<%inherit file="/base.mako"/>
```

Comment:
(merge with -dev didn't resolve this -- opening separate PR to strip it out (again))